### PR TITLE
Updated Pablo Pool Migration in Picasso - Added PICA/KSM pool [CU-861m40629]

### PR DESCRIPTION
**Pull request by PoisonPhang**
[[CU-861m40629]](https://app.clickup.com/t/861m40629)
Updated the initial pool configuration for Pablo via the Picasso runtime migration to add a PICA/KSM pool

### CODE REVIEW

File: code/parachain/runtime/picasso/src/migrations.rs [hunk 1]
```diff
@@ -145,6 +145,13 @@ pub mod pablo_picasso_init_pools {
 					CurrencyId::PICA_USDT_LPT,
 					Permill::from_rational::<u32>(3, 1000),
 				),
+				PoolCreationInput::new_two_token_pool(
+					CurrencyId::PICA,
+					Permill::from_percent(50),
```
Comment (dzmitry-lahoda) [inline code/parachain/runtime/picasso/src/migrations.rs:150]: just to be sure, open/seeding pool will not require 50/50 tokens? so PICA != KSM? just double checking.

**Review by dzmitry-lahoda — APPROVED** [review]

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-861m40629 Support initial pool PICA/KSM](https://app.clickup.com/t/861m40629)

**Review by vimukthi-git — APPROVED** [review]
LGTM

File: code/parachain/runtime/picasso/src/migrations.rs (hunk 1 above)
Comment (PoisonPhang) [reply code/parachain/runtime/picasso/src/migrations.rs:150]: Me and @benluelo are ensuring this via `add_liquidity` in another task

**Review by KaiserKarel — APPROVED** [review]

**Pull request merged**